Src/Support/Google.Apis.Auth/OAuth2/Requests/TokenRequestExtenstions.cs

```diff
@@ -42,25 +41,20 @@ public static class TokenRequestExtenstions
         /// <see cref="Google.Apis.Auth.OAuth2.Responses.TokenResponse.Issued"/> property.
         /// </param>
         /// <returns>Token response with the new access token.</returns>
-        public static async Task<TokenResponse> ExecuteAsync(this TokenRequest request, HttpClient httpClient,
-            string tokenServerUrl, CancellationToken taskCancellationToken, IClock clock)
-        {
-            var httpRequest = new HttpRequestMessage(HttpMethod.Post, tokenServerUrl);
-            httpRequest.Content = ParameterUtils.CreateFormUrlEncodedContent(request);
-
-            var response = await httpClient.SendAsync(httpRequest, taskCancellationToken).ConfigureAwait(false);
-            var content = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+        public static Task<TokenResponse> ExecuteAsync(this TokenRequest request, HttpClient httpClient,
+            string tokenServerUrl, CancellationToken taskCancellationToken, IClock clock) =>
+            ExecuteAsync(request, httpClient, tokenServerUrl, clock, ApplicationContext.Logger, taskCancellationToken);
 
-            if (!response.IsSuccessStatusCode)
+        internal static async Task<TokenResponse> ExecuteAsync(this TokenRequest request, HttpClient httpClient,
```

Comment:
Why is it a good idea to have an internal method with the same parameters but in a different order? That feels like a recipe for confusion to me. If the idea is just to add the logger, I'd keep all the other parameters in the same order and add the new one at the end. (I know it's a bit odd not to have the cancellation token at the end, but I think we can make an exception here.)